integration/run.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/sh -xe
+
+# update project
+rm -r /tmp/rmt-server/*
+cp -r /tmp/workdir/* /tmp/rmt-server
```

Comment:
I think this copies over the `public` directory to the container and makes it very big. I mirrored some repos before in development mode there and then it seems like the docker container was very big because the folder has been copied over. 

This can also be resolved by keeping the development folder clean but apparently the same error does not appear when I run integration tests in master branch.

So I am not sure - maybe we can skip some folder while copying or just mount the volume instead?

Reply:
ok I think it is more issue of my setup - I should have kept the development folder clean and not clutter that with mirrored repo data, so no changes need to the PR in that context